res/doc

```diff
@@ -79,6 +80,12 @@
 #                                      reading_life:words        - My Words
 #                                      store:overdrive           - OverDrive (4.10.11655+) (note: if you don't have an active OverDrive account, it will give you a "Network Error")
 #                                      store:search              - Search
+#                   nickel_wifi    - one of:
+#                                      autoconnect        - attempts to enable and connect to wifi (similar to what happens when you open a link)
+#                                      autoconnect_silent - attempts to connect to wifi in the background (does nothing if wifi is disabled, the battery is low, is already connected, or there aren't any known networks in range) (no errors are shown) (similar to what happens when you turn on the Kobo)
+#                                      enable             - enables WiFi (but doesn't necessarily connect to it)
+#                                      disable            - disables WiFi (but doesn't necessarily connect to it)
```

Comment:
The comment between parentheses doesn't make sense for disable ;).

Reply:
:facepalm:

Reply:
Fixed in 1001be39aaf25f54edfd67b97103c0ab23ce59fd.